src/TuDa.CIMS.Shared/Dtos/CreatePurchaseDto.cs

```diff
@@ -0,0 +1,39 @@
+﻿using TuDa.CIMS.Shared.Entities;
+
+namespace TuDa.CIMS.Shared.Dtos;
+
+public abstract record CreatePurchaseDto
```

Comment:
🔧 Some things are not used in `CreateAsync` of the Service

Reply:
Maybe we should talk about this because I don't understand what do you mean with this comment.